src/envs/behavior_options.py

```diff
@@ -461,46 +462,160 @@ def create_grasp_policy(
 
     def graspObjectOptionPolicy(_state: State,
                                 env: "BehaviorEnv") -> Tuple[Array, bool]:
+        nonlocal plan
+        nonlocal reversed_plan
         nonlocal plan_executed_forwards
         nonlocal tried_closing_gripper
         done_bit = False
-        if (not plan_executed_forwards and not tried_closing_gripper):
-            # Step thru the plan to execute Grasping
-            # phases 1 and 2
-            ret_action = get_delta_low_level_hand_action(
-                env,
-                plan[0][0:3],
-                plan[0][3:6],
-                plan[1][0:3],
-                plan[1][3:6],
-            )
-            plan.pop(0)
-            if len(plan) == 1:
+
+        atol_xyz = 1e-4
+        atol_theta = 0.1
+        atol_vel = 5e-3
+
+        # 1. Get current position and orientation
+        current_pos, current_orn_quat = p.multiplyTransforms(
+            env.robots[0].parts["right_hand"].parent.parts["body"].new_pos,
+            env.robots[0].parts["right_hand"].parent.parts["body"].new_orn,
+            env.robots[0].parts["right_hand"].local_pos,
+            env.robots[0].parts["right_hand"].local_orn,
+        )
+        current_orn = p.getEulerFromQuaternion(current_orn_quat)
+
+        if (  # pylint:disable=no-else-return
```

Comment:
what's going on with this pylint?

Reply:
pylint doesn't like there being an 'elif' after an 'if' with a return statement, but i'm not sure if there's a better way to organize the logic (lmk if you see one!). Will add a comment explaining this though

Reply:
just change the elif to an if (and un-indent any else if you have one)

Reply:
pylint is just saying that there's no point using `elif` bc if the `if` statement is run, it's guaranteed that the function will `return` somewhere inside it